components/esp_http_client/esp_http_client.c

```diff
@@ -1155,6 +1155,18 @@ int esp_http_client_get_errno(esp_http_client_handle_t client)
     return esp_transport_get_errno(client->transport);
 }
 
+esp_err_t esp_http_client_get_tls_error(esp_http_client_handle_t client, int *tls_code, int *tls_flags){
```

Comment:
Could you please API name to esp_http_client_get_and_clear_last_tls_error?

Reply:
done